meta-refkit-core/recipes-connectivity/linux-intel/linux-intel_%.bbappend

```diff
@@ -0,0 +1,2 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+SRC_URI_append = " file://nftables.cfg"
```

Comment:
yocto-kernel-cache has nf_tables KERNEL_FEATURE. Have you submitted these in yocto-kernel-cache yet? Or would the existing nf_tables be enough?